contrib/subrepo-cheri-c-tests/test_runtime.c

```diff
@@ -115,7 +115,7 @@ enum cheri_branch_ops
 static inline struct cheri_frame *
```

Comment:
This code also seems to be missing CHERI_BRANCH_CBEZ (0x11) and CHERI_BRANCH_CBEZ (0x12).
However, I don't think there is any need to add them since this signal handler is broken and uses the old encoding for CJALR and CJR.

Reply:
I suspect this code is old and just hasn't been updated in a long time.  I wonder if these bits are even exercised anymore (as in, do any of the tests trigger these cases)?

Reply:
In theory the trapping instructions could be moved into a cjr/cjalr branch delay slot but it's unlikely since there is probably something else that can go there in most cases.